CONTRIBUTING.md

```diff
@@ -139,7 +139,7 @@ The fields `name:`, `url:`, `img:`, `tfa:` are required for all entries.
 If a site does provide TFA, it is strongly recommended that you add the `doc:`
 field where public documentation is available. Other fields should be included
 if the website supports them. Any services that are not supported can be excluded.
-Sites supporting TFA should not have a `twitter`, `facebook` or `email_address` field.
+Sites supporting TFA should not have a `twitter:`, `facebook:` or `email_address:` field.
```

Comment:
```suggestion
Sites supporting TFA must not have a `twitter:`, `facebook:` or `email_address:` field.
```
As the checks fail completely (no warning, error) when 2fa is enabled and one these is present I think the language should be a bit more direct.